### PR TITLE
doc/userguide: provide more RPM doc - v1

### PR DESCRIPTION
**Pull request by jasonish**
- Address the various RPM distributions
- User info
- Systemd info

Related issue: https://redmine.openinfosecfoundation.org/issues/5884

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -147,27 +147,81 @@ For Debian 10 (buster), for instance, run the following as ``root``::
     apt-get update
     apt-get install suricata -t buster-backports
 
-Fedora
-^^^^^^
+CentOS, AlmaLinux, RockyLinux, Fedora, etc
```
Comment (jasonish) [inline doc/userguide/install.rst:150]: Large enough section go have its own page? I'm thinking so. I'd like to see similar for Debian packages as well.

Comment (jufajardini) [reply doc/userguide/install.rst:150]: Wouldn't hurt, imo.

**Review by jufajardini — COMMENTED** [review]
Changes seem very welcome, to me. Also in favour of having a separate page.

Two comments, one is something I truly don't know, other is a nit.

File: doc/userguide/install.rst [hunk 2]
```diff
@@ -147,27 +147,81 @@ For Debian 10 (buster), for instance, run the following as ``root``::
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RPMs are provided for the latest released of *Enterprise Linux*. This
+includes CentOS 7 and rebuilds such as AlmaLinux and
```
[the remark this answers is not in the document]
Comment (jasonish) [reply doc/userguide/install.rst:154]: Yeah, probably.

**Comment by jufajardini** [comment]
Continued in https://github.com/OISF/suricata/pull/8853

**Pull request closed without merging**